### PR TITLE
Ballerina overview doc

### PR DESCRIPTION
**Pull request by Asitha**
resolves #

### CODE REVIEW

**Comment by Asitha** [comment]
@samgnaniah Ballerina overview doc is partially done. Hence I only added the platform overview part and removed any navigation to it. You can

File: docs/content/src/references/reintroducing-ballerina.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# Re-introducing Ballerina Programming Language
+
+Why does the Ballerina programming language need a re-introduction? You might be a total stranger to 
+the Ballerina programming language, or you have tried it in the past and now you want to try out the latest 
+jBallerina 1.0 release with WSO2 Ballerina Integrator. 
+
+In either case, we are going to skim through the most important parts of the language that you need to know 
+before you write Ballerina programs. If you want in depth learning material you can always refer
```
Comment (samgnaniah) [inline docs/content/src/references/reintroducing-ballerina.md]:
```suggestion
before you write Ballerina programs. If you want in-depth learning material you can always refer to 
```

File: docs/content/src/references/reintroducing-ballerina.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    - [Standard Library](#standard-library)
+      - [The Lang Library](#the-lang-library)
+      - [Ballerina Platform Library](#ballerina-platform-library)
+    - [Ballerina Central](#ballerina-central)
+    - [Ballerina CLI Tool](#ballerina-cli-tool)
+    - [LSP Implementation](#lsp-implementation)
+
+## Introduction to Ballerina Platform
+
+Overall Ballerina platform consists of following key components
```
Comment (samgnaniah) [inline docs/content/src/references/reintroducing-ballerina.md]:
```suggestion
Overall Ballerina platform consists of following key components.
```

File: docs/content/src/references/reintroducing-ballerina.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+- ballerina/lang.int
+- ballerina/lang.string
+- ballerina/lang.array
+- ballerina/lang.map
+- ballerina/lang.future
+
+#### Ballerina Platform Library
+
+All the other modules in the standard library are versioned and released under independent of
```
Comment (samgnaniah) [inline docs/content/src/references/reintroducing-ballerina.md]:
```suggestion
All the other modules in the standard library are versioned and released independent of 
```

File: docs/content/src/references/reintroducing-ballerina.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+### Ballerina Central
+
+[Ballerina Central](https://central.ballerina.io/) is a public module repository maintained by the Ballerina language
+team. You can use this repository to share your Ballerina modules or to use publicly shared Ballerina modules. All the
+WSO2 Ballerina Integrator related connectors are written as Ballerina modules and shared through Ballerina Central.
+
+### Ballerina CLI Tool
+
+Ballerina CLI tool is independent from all the other components mentioned earlier. You can think of this as a
+CLI interface to interact with afore mentioned components of the Ballerina echo system.
```
Comment (samgnaniah) [inline docs/content/src/references/reintroducing-ballerina.md]:
```suggestion
CLI interface to interact with afore mentioned components of the Ballerina ecosystem. 
```

File: docs/content/src/references/reintroducing-ballerina.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+[Ballerina Central](https://central.ballerina.io/) is a public module repository maintained by the Ballerina language
+team. You can use this repository to share your Ballerina modules or to use publicly shared Ballerina modules. All the
+WSO2 Ballerina Integrator related connectors are written as Ballerina modules and shared through Ballerina Central.
+
+### Ballerina CLI Tool
+
+Ballerina CLI tool is independent from all the other components mentioned earlier. You can think of this as a
+CLI interface to interact with afore mentioned components of the Ballerina echo system. 
+
+For instance, you can use this tool to
```
Comment (samgnaniah) [inline docs/content/src/references/reintroducing-ballerina.md]:
```suggestion
For instance, you can use this tool to:
```

File: docs/content/src/references/reintroducing-ballerina.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+- compile your Ballerina programs
+- run unit tests
+- generate API docs
+- push modules to Ballerina central
+- pull modules from Ballerina central
+- Manage multiple Ballerina versions
+
+### LSP Implementation
+
+Ballerina language support different IDE like VS code and IntelliJ Idea through its language server
```
Comment (samgnaniah) [inline docs/content/src/references/reintroducing-ballerina.md]:
```suggestion
Ballerina language supports different IDEs like VS Code and IntelliJ Idea through its language server 
```

File: docs/content/src/references/reintroducing-ballerina.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+- run unit tests
+- generate API docs
+- push modules to Ballerina central
+- pull modules from Ballerina central
+- Manage multiple Ballerina versions
+
+### LSP Implementation
+
+Ballerina language support different IDE like VS code and IntelliJ Idea through its language server 
+Implementation.
```
Comment (samgnaniah) [inline docs/content/src/references/reintroducing-ballerina.md]:
```suggestion
implementation.
```

Shall we recommend VS Code here?

**Pull request merged**